pyccel/parser/syntactic.py

```diff
@@ -1034,6 +1034,7 @@ def _visit_CommentMultiLine(self, stmt):
 
                 txt = com[1:].lstrip()
                 exprs.append(Comment(txt))
+            exprs[-1].set_fst(stmt)
```

Comment:
Is this a bug fix?

Reply:
A very small one yes. I have added it to the PR description